cg/constants/nanopore_files.py

```diff
@@ -0,0 +1,19 @@
+from enum import StrEnum
+from pathlib import Path
+
+
+class NanoporeDirsAndFiles(StrEnum):
+    data_directory: str = "nanopore"
+    sequencing_summary_pattern: str = r"final_summary_*.txt"
+
+    @classmethod
+    def sequencing_summary_file(cls, flow_cell_directory: Path) -> Path | None:
+        try:
+            file = Path(next(flow_cell_directory.glob(cls.sequencing_summary_pattern.value)))
+        except StopIteration:
+            file = None
+        return file
+
+    @classmethod
+    def sequencing_complete(cls, flow_cell_directory: Path) -> bool:
+        return bool(cls.sequencing_summary_file(flow_cell_directory))
```

Comment:
Maybe weird to have this in an Enum class. Any suggestions on how to create a constant for a file containing the flow cell in the name?

Reply:
Given that it is not really used as an enum (?), maybe it makes more sense to have it as a BaseModel with the Path and bool as fields and an init method setting them? Haven't read through the whole thing so I might be misinterpreting it.

Reply:
You can have a function that generates an f string with certain input parameters I suppose.

Reply:
Moved the two class methods to utils for now